power-the-polls-form/tsconfig.json

```diff
@@ -25,7 +25,7 @@
    },
    "include": [
       "src",
-      "types/jsx.d.ts"
+      "types/jsx.d.ts", "../scripts/github-actions/getFullJurisdictions.ts"
```

Comment:
VSCode auto-add?

Reply:
yep, it is from a branch that I pull off from this work. Good catch.